src/v2/coins/abstractUtxoCoin.js

```diff
@@ -1016,14 +1016,14 @@ class AbstractUtxoCoin extends BaseCoin {
         return keyArray.map((k) => k.derive(index));
       }
 
-      const queryBlockchainUnspentsPath = co(function *queryBlockchainUnspentsPath(keyArray, basePath) {
+      const queryBlockchainUnspentsPath = co(function *queryBlockchainUnspentsPath(keyArray, basePath, segwit) {
```

Comment:
You don't need to take a segwit flag. You can get the chain (and thus the address type) from the basepath.